drivers/dac/dac_dacx0501.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2024 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief Driver for Texas Instruments DACx0501 series
+ *
+ * Device driver for the  Texas Instruments DACx0501 series of devices: DAC60501, DAC70501 and
```

Comment:
I would move this description to a YAML file, but let's hear what others think about it.

Reply:
@martinjaeger Can you address @bbilas comment?